xbmc/pvr/dialogs/GUIDialogPVRTimerSettings.cpp

```diff
@@ -284,15 +275,9 @@ void CGUIDialogPVRTimerSettings::InitializeSettings()
   if (tag->SupportsFolders())
     AddEdit(group, SETTING_TMR_DIR, 19076, 0, tag->m_strDirectory, true, false, 19104);
 
-  AddToggle(group, SETTING_TMR_RADIO, 19077, 0, tag->m_bIsRadio);
```

Comment:
so we'de never be allowed to remove obsolete strings? These are core strings, what do we care about add-ons? If an add-on can reuse a core string, fine, but we should not keep strings only because one out of 1000 add-ons might use it.